spring.core/src/main/java/com/epam/springcore/task/facade/GymFacade.java

```diff
@@ -1,70 +1,133 @@
 package com.epam.springcore.task.facade;
 
-
+import com.epam.springcore.task.dto.PassUsernameDTO;
+import com.epam.springcore.task.dto.TraineeDTO;
+import com.epam.springcore.task.dto.TrainerDTO;
+import com.epam.springcore.task.dto.TrainingDTO;
 import com.epam.springcore.task.model.Trainee;
-import com.epam.springcore.task.model.Trainer;
-import com.epam.springcore.task.model.Training;
-import com.epam.springcore.task.service.TraineeService;
-import com.epam.springcore.task.service.TrainerService;
-import com.epam.springcore.task.service.TrainingService;
+import com.epam.springcore.task.service.impl.TraineeService;
+import com.epam.springcore.task.service.impl.TrainerService;
+import com.epam.springcore.task.service.impl.TrainingService;
+import jakarta.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import java.time.LocalDate;
 import java.util.List;
 import java.util.Optional;
+import java.util.Set;
 
 @Component
-public class GymFacade{
+@Slf4j
+public class GymFacade {
```

Comment:
In the current implementation, this facade is non-sense, in production code it's better skip it